### PR TITLE
Added Automating code for Google with Python - Issue #94

### PR DESCRIPTION
**Pull request by kritikaparmar-programmer**
Added code for automating Google. 😀

### CODE REVIEW

File: System-Automation-Scripts/Automating_Google.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from selenium import webdriver
+from selenium.webdriver.common.keys import Keys
+import time
+from time import sleep
+from webdriver_manager.chrome import ChromeDriverManager
+ 
+driver = webdriver.Chrome(ChromeDriverManager().install())
+
+# enter the link of the google
+driver.get("http://www.google.com")
```
Comment (Neilblaze) [inline System-Automation-Scripts/Automating_Google.py]: Webdriver prioritize `https` more than `http`. Please update the following changes in the parse link!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (Neilblaze) [inline .vscode/settings.json]: Remove this json file.

**Review by Neilblaze — CHANGES_REQUESTED** [review]

**Comment by Neilblaze** [comment]
Thank you for your contribution @kritikaparmar-programmer !

Before I approve changes to the PR, I will request you the following changes mentioned below.

- Add README (guide) along with this inside the folder same as your `<filename>`
- Update the README with appropriate installation methods and description guidelines.
- Maintain proper [Contribution protocols](https://github.com/ankitdobhal/Awesome-Python-Scripts/blob/master/Contribution.md) (ignore if you're already following it!) & [good practices](https://gist.github.com/sloria/7001839)!
~~- Recheck if there are any intendation errors in the script.~~

Rest LGTM :+1:

**Comment by kritikaparmar-programmer** [comment]
Hey @Neilblaze , I didn't understand where I've to add Readme, the first point you mentioned.. Can you help with that please

**Comment by powerexploit** [comment]
> Hey @Neilblaze , I didn't understand where I've to add Readme, the first point you mentioned.. Can you help with that please

@kritikaparmar-programmer Nice contributions.
- I suggest you to create folder as Googler inside System-Automation-folder .
- Then add your script and in README add some explanation about your script. 
Thanks

File: System-Automation-Scripts/Googler/README.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# System-Automation-Scripts
+## Automating Google using Selenium
+
+#### Install Requirements : 
+1. Install Selenium by writing the following pip command, at command prompt - 
+> pip install selenium
+
+2. Install ChromeDriver
+
+### Python Scripts in this folder
```
Comment (powerexploit) [inline System-Automation-Scripts/Googler/README.md]: Why this line remove it from here.

**Review by powerexploit — COMMENTED** [review]
Before I merged it remove extra lines from README.

File: System-Automation-Scripts/Googler/README.md (hunk 3 above)
Comment (kritikaparmar-programmer) [reply System-Automation-Scripts/Googler/README.md]: Updated it...

Comment (Neilblaze) [reply System-Automation-Scripts/Googler/README.md]: LGTM :+1:

**Review by Neilblaze — APPROVED** [review]
Nice work @kritikaparmar-programmer ! :tada:

**Pull request merged**